apps/main/src/lend/lib/apiLending.ts

```diff
@@ -395,21 +398,30 @@ const user = {
           console.error('Failed to fetch user loss:', error)
         }
 
-        const resp = await market.stats.bandsInfo()
-        const { liquidationBand } = resp ?? {}
+        let liquidationBand = null
```

Comment:
the fact you need `let` here probably means this part should be a separate function with early return